### ISSUE TITLE
Finetuning with limited computational resources

### ISSUE THREAD

**Issue by PixarHero** [Code-kunkun/LamRA#16]
Thank you for your excellent work! I would like to inquire about fine-tuning with limited computational resources. Specifically, I have access to 2 A100 GPUs with 80GB of memory each. Given these constraints, I would like to know if fine-tuning is feasible and if you have any advice for optimizing the process. Can using a small batch size lead to poor model performance or result in a loss of zero or  is there a minimum batch size threshold below which training fails to work? Thx!

**Comment by Code-kunkun** [comment]
Thank you for your attention. Training with 2 A100 GPUs might be quite challenging because the batch size has a significant impact on the results. However, you could consider using gradient accumulation to alleviate the batch size issue, though I'm not sure how effective it will be.
